.github/workflows/dev.yml

```diff
@@ -0,0 +1,29 @@
+name: Check for New DEV Interactions and Add to Orbit
+on:
+  schedule:
+    - cron: "0 0 */1 * *"
+  workflow_dispatch:
+jobs:
+  comments-workflow:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        submodules: recursive
+    - name: Set up Ruby 2.7.2
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7.2
+    - name: Bundle Install
+      run: |
+        gem update --system 3.1.4 -N
+        gem install --no-document dev_orbit
+    - name: Check for New Comments for a DEV Organization
+      run: |
+        dev_orbit --check-organization-comments --historical-import
+      env:
+        DEV_API_KEY: ${{ secrets.DEV_API_KEY }}
+        DEV_ORGANIZATION: ${{ secrets.DEV_ORGANIZATION }}
```

Comment:
I know ~nothing about this stuff, but just glancing through the [README](https://github.com/bencgreenberg/dev_orbit#instantiation-of-the-client), should this maybe be `DEV_USERNAME`?

Reply:
Good question! Since we're working with an org on Dev.to, we should keep `DEV_ORGANIZATION`. `DEV_USERNAME` is for if you're pulling down your personal activities :)

Reply:
Ohhh, interesting that the old repo is what comes up in search. The current one is https://github.com/orbit-love/community-ruby-dev-orbit#instantiation-of-the-client